CSS/styles.css

```diff
@@ -0,0 +1,434 @@
+/* ================================ Base styles ================================ */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    -webkit-font-smoothing: antialiased; /* do I need this? */
+    background: #FFF;
+    color:#222;
+}
+
+ul {
+    list-style: none;
+}
+
+/* 
+h1, h2, h3, h3, h4 {
+
+}
+*/
+
+a {
+    color: #222;
+    text-decoration: none;
+}
+
+
+p {
+    margin: 0.5rem 0;
+}
+
+img {
+    width: 100%;
+}
+
+.content-wrap {
+    width: 90%;
+    margin: 0 auto;
+}
+
+/* Buttons */
+
+button {
+    background-color: transparent;
+    border: none;
+}
+
+.btn {
+    display: inline-block;
+    cursor: pointer;
+    border: 1px solid #DDD;
+    background-color: #FFF;
+}
+
+.btn:hover {
+    box-shadow: rgba(0, 0, 0, 0.07) 0px 4px 6px;
+}
+
+.btn-rounded {
+    border-radius: 4px;
+}
+
+/* ================================ Header ================================ */
+/* ---------------------------- Header Top Left --------------------------- */
+header {
+    width: 100%;
+    height: 178px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 100;
+    background-color: #FFF;
+}
+
+.header-top {
+    border-bottom: 1px solid #DDD;
+}
+
+.header-top-container {
+    height: 80px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.header-logo img {
+    width: 107px;
+}
+
+/* ---------------------------- Header Top Middle --------------------------- */
+.header-button-container {
+    display: flex;
+    height: 48px;
+    text-align: center;
+    border-radius: 30px;
+    align-items: center;
+}
+
+.header-button {
+    padding: 14px 18px 19px;
+    font-size: 15px;
+}
+
+.button-separator-line {
+    flex: 0 0 1px;
+    height: 24px;
+    width: 1px;
+    background-color: #DDD;
+}
+
+/* ---------------------------- Header Top Right --------------------------- */
+.header-right-container {
+    display: flex;
+    align-items: center;
+}
+
+.become-host-link {
+/*    width: 123px;*/
+    height: 42px;
+    border-radius: 30px;
+    border: none;
+    padding: 14px 15px 16px;
+    font-size: 14px;
+}
+
+.become-host-link:hover {
+    background-color: #F7F7F7;
+}
+
+.choose-language {
+    width: 40px;
+    height: 42px;
+    padding: 13px;
+    margin-left: -7px;
+}
+
+.choose-language img {
+    width: 16px;
+    height: 16px;
+}
+
+.choose-language:hover {
+    background-color: #F7F7F7;
+    border-radius: 22px;
+}
+
+.signup {
+    width: 77px;
+    height: 42px;
+    border: 1px solid #DDD;
+    border-radius: 30px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+img#menu {
```

Comment:
These types of selectors are discouraged. The ID should be sufficient as there should only be one ID in the page and it doesn't matter what element it is on. Also, IDs are often discouraged and CSS classes are they way to go.

Reply:
I don't remember why I did that, but I've changed it to classes and it seems to work fine.